pkg/envoy/resources.go

```diff
@@ -14,7 +14,17 @@
 
 package envoy
 
-import "github.com/cilium/cilium/pkg/envoy/xds"
+import (
+	"sort"
+
+	envoyAPI "github.com/cilium/cilium/pkg/envoy/cilium"
```

Comment:
What is wrong with the `cilium` package name? :)

Reply:
copy+paste :-)